docs/cookbook-recipes/isle-v140-git-cleanup.md

```diff
@@ -78,7 +78,8 @@ Enter the name here:
 * The script will continue and the process takes about 30 - 45 seconds depending on the speed of your system with minimal output.
 
 * Your final step is to push this local smaller repo back to the remote repository
-* `git push origin master`
+  * `cd ISLE.git` or `cd yourprojectnamehere-isle.git`
+  * `git push origin master`
 
 * Clone down your newly changed and smaller forked or cloned ISLE project directory again to ensure that the repositiory is smaller to a location of your choice.
```

Comment:
I think we to say something here to the effect:  *Do not work out of previously cloned git repositories.* You will need to remove any previously existing clones of this repository and clone down new ones.